py/desisim/test/test_obs.py

```diff
@@ -119,12 +119,25 @@ def test_newexp_sky(self):
         sim_dark, fmap_dark, meta_dark, obscond_dark, objmeta_dark = obs.new_exposure('dark', nspec=10, night=night, expid=0, exptime=1000)
         dark = sim_dark.simulated.copy()
         sim_mws, fmap_mws, meta_mws, obscond_mws, objmeta_mws = obs.new_exposure('mws', nspec=10, night=night, expid=1, exptime=1000)
-        mws = sim_dark.simulated.copy()
+        mws = sim_mws.simulated.copy()
         for channel in ['b', 'r', 'z']:
             sky_mws = mws['num_sky_electrons_'+channel]
             sky_dark = dark['num_sky_electrons_'+channel]
             nonzero = (sky_mws != 0.0)
-            self.assertTrue(np.all(sky_mws[nonzero] > sky_dark[nonzero]))
+            try:
+                self.assertTrue(np.all(sky_mws[nonzero] > sky_dark[nonzero]))
+            except:
+                import matplotlib.pyplot as plt
+                ww = np.where(sky_mws[:, 0] < sky_dark[:, 0])[0]
+                plt.clf()
+                plt.plot(dark['wavelength'][ww], sky_dark[ww, 0], label='dark')
+                plt.plot(dark['wavelength'][ww], sky_mws[ww, 0], alpha=0.5, label='MWS')
+                plt.xlabel('Wavelength (Angstrom)')
+                plt.ylabel('Simulated sky (electron)')
+                plt.legend()
+                plt.subplots_adjust(left=0.2)
+                plt.show()
+                import pdb ; pdb.set_trace()
```

Comment:
Units stuff looks ok, but let's not leave plotting and `pdb.set_trace()` within the unit tests (or else travis could give us mystery hangs etc.)